action.yml

```diff
@@ -35,16 +35,16 @@ runs:
       shell: bash
       working-directory: ${{ github.action_path }}
       run: |
-        kernel-crawler crawl --distro=${{ inputs.distro }} --arch=${{ inputs.arch }} > kernels_${{ inputs.arch }}.json
+        kernel-crawler crawl --distro=${{ inputs.distro }} --arch=${{ inputs.arch }} > ${{ runner.temp }}/kernels_${{ inputs.arch }}.json
```

Comment:
This fixes the `test.yml` CI, since before it was failing with:
```

test kernel-crawler
Invalid pattern '/home/runner/work/kernel-crawler/kernel-crawler/.//kernels_x86_64.json'. Relative pathing '.' and '..' is not allowed.
```
because `${{ github.action_path }}` was relative to the root folder.
Using runner temporary folder yelds better results.